config/cockroachdb/sample_ycsb_config.xml

```diff
@@ -16,6 +16,9 @@
     <!-- Optional: Override the field size for each column in USERTABLE -->
     <!-- <fieldSize>8</fieldSize> -->
 
+    <!-- Optional: Override the zipfian constant to modify the skew -->
+    <!-- <zipfianConstant>0.99</zipfianConstant> -->
```

Comment:
More generic / understandable variable name (e.g., `skewness`) is preferred since it can be applied to record selection for other benchmarks such as TPC-C in the future. My 2c.

Reply:
I agree with @stonewhitener on this!

@MohammadKhalaji Can you rename the configuration parameter `skewFactor`?

Reply:
@apavlo Done.

Reply:
I decided to keep it `zipfianConstant` in the `YCSBBenchmark` itself but rename it to `skewFactor` in the config files. Because in YCSB, the distribution is explicitly zipfian.